components/Navbar/Rightnav.js

```diff
@@ -0,0 +1,82 @@
+import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
+
+const Ul = styled.ul`
```

Comment:
Maybe you could name this for example "StyledUl", so it would be semantically more clear for you? :)